.github/workflows/test_pyfans.yaml

```diff
@@ -0,0 +1,37 @@
+name: Test PyFans
+
+on: [push, pull_request]
+
+jobs:
+  pyfans-run-test:
+    runs-on: ubuntu-latest
+    container: unistuttgartdae/fans-ci:noble
+    defaults:
+      run:
+        shell: "bash --login -eo pipefail {0}"
+    env:
+      FANS_BUILD_DIR: build
+      FANS_MPI_USER: fans
+    steps:
+
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Generate build directory
+      run: mkdir -p ${{ env.FANS_BUILD_DIR }}
+
+    - name: Install dependencies
+      run: |
+        apt update
+        apt install -y cmake make g++ python3 python3-numpy python3.12-dev
+
+    - name: Configure
+      working-directory: ${{ env.FANS_BUILD_DIR }}
+      run: |
+        cmake .. -DFANS_LIB=ON
+        make
```

Comment:
Maybe add a `-j` here for parallel compilation? However it would make error messages less easy to read due to the simultaneous outputs...

Reply:
The compilation does not take too long, so I would not put `-j` here.